homeassistant/components/light/__init__.py

```diff
@@ -448,7 +452,13 @@ async def async_handle_light_on_service(light, call):
                 params[ATTR_HS_COLOR] = color_util.color_RGB_to_hs(*rgb_color)
             elif (rgbww_color := params.pop(ATTR_RGBWW_COLOR, None)) is not None:
                 rgb_color = color_util.color_rgbww_to_rgb(
-                    *rgbww_color, light.min_mireds, light.max_mireds
+                    rgbww_color[0],
+                    rgbww_color[1],
+                    rgbww_color[2],
+                    rgbww_color[3],
+                    rgbww_color[4],
```

Comment:
I don't think that is better. Instead of just the opcodes `LOAD_GLOBAL` and `CALL_FUNCTION_EX`, Python would have to do **three for each argument** `BINARY_SUBSCR`, `LOAD_GLOBAL`, and `LOAD_CONST`.

Typing those correctly is a bit more challenging. Let's add `type: ignores` for now and come back later (if necessary).